app/code/Magento/Catalog/Ui/DataProvider/Product/ProductDataProvider.php

```diff
@@ -52,12 +60,14 @@ public function __construct(
         array $addFieldStrategies = [],
         array $addFilterStrategies = [],
         array $meta = [],
-        array $data = []
+        array $data = [],
+        PoolInterface $pool = null
```

Comment:
I would suggest using a more self-explanatory argument/property name here. For example `$modifiersPool`. Since `$pool` is a very generic name and might be confusing

Reply:
@rogyar done.